server/src/game/index.js

```diff
@@ -119,14 +159,8 @@ export class Game {
     };
   }
 
-  broadcastPosition(user) {
-    server.io.to(this.id).emit(Events.UPDATE_POSITION, {
-      id: user.id,
-      x: user.x,
-      y: user.y,
-      vx: user.vx,
-      vy: user.vy
-    });
+  broadcastPosition({ id, x, y, vx, vy}, event = Events.UPDATE_POSITION) {
+    server.io.to(this.id).emit(event, { id, x, y, vx, vy });
```

Comment:
how do you know if it's an user id or a mob id?

Reply:
You can pass another Event in argument like `Events.UPDATE_MOB`.